engine/api/workflow/dao_run_test.go

```diff
@@ -109,7 +109,7 @@ func TestPurgeWorkflowRun(t *testing.T) {
 	test.NoError(t, err)
 
 	for i := 0; i < 5; i++ {
-		_, _, errWr := workflow.ManualRun(nil, db, db, cache, proj, w1, &sdk.WorkflowNodeRunManual{
+		_, _, errWr := workflow.ManualRun(nil, db, cache, proj, w1, &sdk.WorkflowNodeRunManual{
```

Comment:
do not pass a nil Context, even if a function permits it; pass context.TODO if you are unsure about which Context to use